code/modules/organs/organ_icon.dm

```diff
@@ -159,12 +159,10 @@ var/global/list/limb_icon_cache = list()
 					icon_cache_key += "[M][markings[M]["color"]]"
 
 			if(body_hair && islist(h_col) && h_col.len >= 3)
-				var/cache_key = "[body_hair]-[icon_name]-[h_col[1]][h_col[2]][h_col[3]]"
-				if(!limb_icon_cache[cache_key])
-					var/icon/I = icon(species.get_icobase(owner), "[icon_name]_[body_hair]")
-					I.Blend(rgb(h_col[1],h_col[2],h_col[3]), ICON_MULTIPLY) //VOREStation edit
-					limb_icon_cache[cache_key] = I
-				mob_icon.Blend(limb_icon_cache[cache_key], ICON_OVERLAY)
+				var/icon/I = icon(species.get_icobase(owner), "[icon_name]_[body_hair]")
+				I.Blend(rgb(h_col[1],h_col[2],h_col[3]), ICON_MULTIPLY) //VOREStation edit
+				mob_icon.Blend(I, ICON_OVERLAY)
+				icon_cache_key += "[body_hair][rgb(h_col[1],h_col[2],h_col[3])]"
```

Comment:
Um, so you're just entirely removing caching for these?

Reply:
Yeah, it no longer pulls directly from a snowflake cache in `limb_icon_cache` and just uses the same methods that limbs normally use (appendiglng to `icon_cache_key` and getting to `limb_icon_cache` from there.
This section is now treated the exact same as markings

Reply:
-snipped off a bunch of BS- Yeah, I'm taking it off because I'm wondering if this behaviour is related to the crashing issues, nothing more. The caching looks perfectly fine to me.

I could leave it off this PR to see if just fixing the business in `update_icons.dm` does the trick.

Reply:
Is this the only thing that used limb cache...?

Reply:
The only other place `limb_icon_cache` is used is for the base limb icons in health-doll icon generation over at https://github.com/VOREStation/VOREStation/blob/15d55d5d06afc518e1167f639acfcc662468a2bf/code/modules/organs/organ_icon.dm#L240